### DISCUSSION TITLE
FR: Hotkey or context menu item to set the new node layout functionality

### DISCUSSION THREAD

**Discussion by Lotus49b** [freeplane/freeplane#1066] [Ideas]
The new layouts make FP very flexible in building maps. However, going to the Format sidebar and scrolling through all the options to set layouts breaks the flow.

It would be great if the node layout options could be activated directly from a hotkey (maybe a couple of keys to apply the most regularly used layouts), or a similar thing using the right-click context menu.

To apply the style permanently to a node level it might still be necessary to use the Format menu, but that would be okay for that requirement.

Thanks for a great app, the new versions are game-changers.

**Comment by sifran-github** [comment]
I think these tips for quick formatting will solve your problems:

1. use the option: format -> preserve format for new siblings / for new child nodes (I use it for siblings).
2. Must have for quick formatting: just use format->copy format and format->paste format! Its a life saver. I configured special keys for them, and and I always use this option. So basically you just have to set some layouts just one time, and then just copy and paste the style quickly to each new node that you want. you can even copy/past between maps. easy.

**Reply by Lotus49b** [reply]
Thanks! Just tried this and it works great, really speeds things up.
